internal/future/testing/fake_frontend.go

```diff
@@ -18,31 +18,39 @@ package testing
 
 import (
 	"context"
-	"fmt"
 
-	"github.com/GoogleCloudPlatform/open-match/internal/pb"
+	"github.com/GoogleCloudPlatform/open-match/internal/future/pb"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 // FakeFrontend is an empty gRPC handler.
 type FakeFrontend struct {
 }
 
-// NewFakeFrontend creates a new FakeFrontend for tests.
-func NewFakeFrontend() *FakeFrontend {
-	return &FakeFrontend{}
+// CreateTicket will create a new ticket, assign a Ticket id to it and put the
+// Ticket in state storage. It will then look through the 'properties' field
+// for the attributes defined as indices the matchmakaking config. If the
+// attributes exist and are valid integers, they will be indexed. Creating a
+// ticket adds the Ticket to the pool of Tickets considered for matchmaking.
+func (s *FakeFrontend) CreateTicket(ctx context.Context, req *pb.CreateTicketRequest) (*pb.CreateTicketResponse, error) {
+	return &pb.CreateTicketResponse{}, nil
```

Comment:
return status.Error(codes.Unimplemented, "not implemented")

Reply:
This is intentional. We use this in tests.